spec/controllers/oral_history_requests_controller_spec.rb

```diff
@@ -143,40 +143,6 @@
       }
     end
 
-    describe "with old email functionality" do
-      describe "automatic delivery" do
-        let(:work) { create(:oral_history_work, :available_by_request, available_by_request_mode: :automatic)}
-
-        it "emails files" do
-          expect {
-            post :create, params: full_create_params
-          }.to have_enqueued_job(ActionMailer::MailDeliveryJob).with { |class_name, action|
-              expect(class_name).to eq "OralHistoryDeliveryMailer"
-              expect(action).to eq "oral_history_delivery_email"
-          }
-
-          expect(response).to redirect_to(work_path(work.friendlier_id))
-          expect(flash[:success]).to match /We are sending you links to the files you requested/
-        end
-      end
-
-      describe "manual review" do
-        let(:work) { create(:oral_history_work, :available_by_request, available_by_request_mode: :manual_review)}
-
-        it "emails admin, and lets user know" do
-          expect {
-            post :create, params: full_create_params
-          }.to have_enqueued_job(ActionMailer::MailDeliveryJob).with { |class_name, action|
-              expect(class_name).to eq "OralHistoryRequestNotificationMailer"
-              expect(action).to eq "notification_email"
-          }
-
-          expect(response).to redirect_to(work_path(work.friendlier_id))
-          expect(flash[:success]).to match /Your request will be reviewed/
-        end
-      end
-    end
-
```

Comment:
So if I remember right, the old flag functionality SHOULD still work even without this column.  Removing the column ought not to remove ability to switch the feature flag back and forth at this point. So ideally these tests might remain?

Did these tests fail? Is it possibly just a test setup problem (trying to set it up with the email in the old column), and if you just set up the data in the new way the tests as written will still pass?

Reply:
I am not sure why these got mistakenly removed. Glad you caught this!